services/app-api/handlers/banners/delete.test.ts

```diff
@@ -0,0 +1,77 @@
+import { deleteBanner } from "./delete";
+
+import { APIGatewayProxyEvent } from "aws-lambda";
+import { proxyEvent } from "../../utils/testing/proxyEvent";
+import { StatusCodes } from "../../utils/types/types";
+import {
+  NO_KEY_ERROR_MESSAGE,
+  UNAUTHORIZED_MESSAGE,
+} from "../../utils/constants/constants";
+
+jest.mock("../../utils/dynamo/dynamodb-lib", () => ({
+  __esModule: true,
```

Comment:
Is this line necessary for all the util mocks?

Reply:
Thank you for making me learn what this is. You are right. We don't need it.
ref: https://jestjs.io/docs/jest-object#jestmockmodulename-factory-options